docs/misc/downloads-api.mdx

```diff
@@ -0,0 +1,63 @@
+---
+slug: /downloads-api
+description: Paper provides a downloads API that you can use to access builds.
+---
+
+# Downloads API
+
+PaperMC provides a downloads API to facilitate automated downloads access. Full documentation can be
+found on the [Downloads API Docs](https://api.papermc.io/docs).
+
+## Getting Highest Stable Build Number
+
+<VersionFormattedCode language={"shell"}>
+```
+#!/bin/bash
+
+PROJECT="paper"
+MINECRAFT_VERSION="%%_MAJ_MIN_MC_%%"
+
+LATEST_BUILD=$(curl -s https://api.papermc.io/v2/projects/${PROJECT}/versions/${MINECRAFT_VERSION}/builds | \
+    jq -r '.builds | map(select(.channel == "default") | .build) | .[-1]')
```

Comment:
I think the `-r` (`output raw strings, not JSON texts`) option can be safely removed here, since the build is an integer

Reply:
I cannot comment on these, i don't know jq syntax and am on my windows PC without wsl. If someone else could confirm please and thank you 🙏 - or if you can test @zlataovce

Reply:
tested, it does work